### PR TITLE
nolo tlc ciel 13_5_1

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 1]
```diff
@@ -991,17 +993,17 @@ Let me out, let me out, let me out...!
...
 -- Page 57, Offset 35417.
 -- お願いします、出してくださいノエル先生……！」
-I'm begging you, let me out of here. Noel-sensei...!"
+I'm begging you, let me out of here, Noel-sensei...!"
 }
 [sha:0412e6585a28640942942c647180dac8abc66596]{
 -- Page 58, Offset 35418.
 -- 精神だけになっても傷はつくのか。
-Can one get hurt even when one is reduced to only their mind?
+So I can still feel pain even though only my mind has been transported here, huh.
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:1001]: pain, even

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 2]
```diff
@@ -1061,12 +1063,12 @@ There isn't a single thing that I can do―――
...
 -- Page 62, Offset 35431.
 -- 「いいえ。そんなに自分を下げちゃダメよー？
-"No. You shouldn't be lowering yourself like that.
+"No. There's no need to be depracating yourself so much, you know?
 }
 [sha:d733d2cb538c92cf1cb91bbc0000c3f9c2985dc3]{
 -- Page 62, Offset 35432.
 -- だって、わたしに飼われているかぎり、できること、やるべきことは沢山あるんだから」
-Because as long as I keep you here like this, there are so many things that you can do, and should do."
+Because as long as you're my pet, there are lots of other things that you can and should be doing."
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:1071]: fata morgana moment

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 3]
```diff
@@ -1091,35 +1093,35 @@ If you lose one finger, then next time, it'll be two.
 [sha:954e160821529104cac0da8c01e8aadd7615698c]{
 -- Page 63, Offset 35437.
 -- 思い出すだけで心が削られる思い出を、少しずつ少しずつ、より悲惨な<過|も><去|の>に変えていって―――」
-Slowly but surely, I'll take those memories that scrapes your heart from just remembering it, and turn it into something even more miserable―――"
+Slowly but surely, I'll take those memories that tear at your heart when you remember them, and turn them into something even more miserable―――"
```
Comment (Avuxo) [inline script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:1096]: remove "when you remember them", it's unnecessary

File: script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt [hunk 4]
```diff
@@ -421,22 +421,22 @@ And that you were far too late, with no help in sight, and that you're just a po
...
 -- Page 24, Offset 35304.
 -- 「……趣味が、悪い、ぜ……
-"...Your tastes, are pretty, messed up...
+"...Your tastes... are pretty... messed up...
 }
 [sha:725ce177305b64a25924c35da00b0782aed03356]{
 -- Page 24, Offset 35305.
 -- 殺すなら、さっさと、殺せ、よ」
-If you're going to kill me, then just, do it."
+If you're going to kill me... then just... do it."
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 13/13_01_02_CIEL13_5_1.txt:434]: took me a while to get this lmao

**Pull request merged**